rust/migrate-wicked/tests/test.sh

```diff
@@ -147,6 +160,44 @@ for test_dir in ${TEST_DIRS}; do
     [ "$NO_CLEANUP" -gt 0 ] || nm_cleanup
 done
 
+if $TEST_STDIN; then
+    echo -e "${BOLD}Testing stdin show${NC}"
+    cat <<EOF | $MIGRATE_WICKED_BIN show --without-netconfig - | grep "192.168.100.5" &>/dev/null
```

Comment:
Why not just running the same tests but simply cutting the xml files to `migrate-wicked`?

Reply:
That was actually my first approach, but I found it a bit annoying to implement and also I think having a separate testcase that you can execute on it's own is better. Plus having it just execute the same testcases really is just duplication, because when the "mechanism" works it does the same thing.

Reply:
okay